google/cloud/forseti/scanner/scanner_builder.py

```diff
@@ -54,10 +54,18 @@ def build(self):
         for scanner in self.scanner_configs.get('scanners'):
             if scanner.get('enabled'):
                 module_path = 'google.cloud.forseti.scanner.scanners.{}'
+
+                requirements_map = scanner_requirements_map.REQUIREMENTS_MAP
+                if not scanner.get('name') in requirements_map:
+                    LOGGER.error('Configured scanner is undefined: %s',
```

Comment:
Looking at this again, it would really help troubleshooting if we add a bit more detail of where it comes from.  Can you try this and see if it works?
```
log_message = (
    'Configured scanner is undefined '
    'in scanner requirements map : %s')
LOGGER.error(log_message, scanner.get('name'))
```